### ISSUE TITLE
When using {@trigger} replies don't seem to be getting eliminated.

### ISSUE THREAD

**Issue by Lewwwk** [superscriptjs/superscript#218]
I am currently using a plugin function to generate some dynamic replies.

```
+ get question
- ^getQuestion()

+ __partone__
- part one reply

+ __parttwo__
- part two reply
```

Then in the plugin I return callback that builds the reply string as follows: `{@__partone__} some other text i dynamically generate {@__parttwo__}`

This functions as expected, EXCEPT that the replies in `__partone__` and `__parttwo__` do not get eliminated when used.

**Comment by silentrob** [comment]
So this is really challenging because we only record ONE reply in the final object and not all three (in your case)

Perhaps we should keep an array of replies and exhaust all of them

**Comment by Lewwwk** [comment]
That makes sense. Not an urgent issue for me, just something I discovered and can work around for now.

**Comment by silentrob** [comment]
Also related, the topicName in the replyObject displays the first match and while that isn't wrong, in some cases you may have a pass though gambit in pre and that will always be the first match, even if that isn't were the bulk of the reply is coming from.

**Comment by benhjames** [comment]
Fixed in v1.0.0.
